website/source/layouts/aws.erb

```diff
@@ -10,6 +10,39 @@
                     <a href="/docs/providers/aws/index.html">AWS Provider</a>
                 </li>
 
+                <li<%= sidebar_current(/^docs-aws-resource-apigateway/) %>>
```

Comment:
This should be `<li<%= sidebar_current(/^docs-aws-resource-api-gateway/) %>>` (with the hyphen)

Otherwise highlighting in the sidebar won't work.